stages/org.osbuild.pki.update-ca-trust

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+import os
+import sys
+import uuid
+
+import osbuild.api
+
+
+def main(tree, options):
+    for cert in options["certs"]:
+        rnd = str(uuid.uuid4())
+        with open(f"{tree}/etc/pki/ca-trust/source/anchors/ostree-{rnd}.crt", "w", encoding="utf8") as f:
+            f.write(cert)
+
+    subprocess.run(["/usr/bin/update-ca-trust", "extract"], check=True)
```

Comment:
Pardon my ignorance here, should this run inside the "tree" e.g. via `subprocess.run(["chroot", tree, ...)` or is it intended like this ?

Reply:
Haven’t realized this, not intentional.